src/Wpf.Ui/Controls/Button/Button.xaml

```diff
@@ -232,6 +233,7 @@
         <Setter Property="CornerRadius" Value="{DynamicResource ControlCornerRadius}" />
         <Setter Property="Appearance" Value="Secondary" />
         <Setter Property="SnapsToDevicePixels" Value="True" />
+        <Setter Property="UseLayoutRounding" Value="True" />
```

Comment:
This is necessary to avoid the border thickness going into gray territory when you're not using 100% display scaling.